ci/github/collect_troubleshooting

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euxo pipefail
+collect_troubleshooting () {
+      kubectl get all -n kmm-operator-system
+      kubectl get all
+      kubectl describe node
+      kubectl describe daemonset -l kmm.node.kubernetes.io/module.name
+      kubectl describe pod -l kmm.node.kubernetes.io/module.name
+      sudo dmesg
```

Comment:
How does that work on an automated flow? Won't it require some user input?

Reply:
And what happens if we drop `sudo`? Will it fail?

Reply:
Yes it fails as the builder user in the worker has no privileges:

```
dpkg: error: requested operation requires superuser privilege
```